flink-table/flink-table-common/src/main/java/org/apache/flink/table/utils/DateTimeUtils.java

```diff
@@ -1290,6 +1353,24 @@ private static long floor(long a, long b) {
         }
     }
 
+    private static long floor(long a, int b) {
```

Comment:
Can we have here and for other methods more selfexplainable names ?
currently it is not clear what `a` and `b` mean here.
At the same time looking at logic like 
```java
long q = b / NANOS_PER_MILLISECOND;
```
it seems it is not a generic floor, so names should have more context

Reply:
I can see that a and b are used elsewhere. I suggest that it could be clearer to use timepoint and timeintervalunit as the variable names to match the documentation. I would also have the full words for q and r. In my view it would be so much more self describing. Though from googling, I see there is a school of thought for maths intensive processing single letters are used - I assume so it looks more like algebra.